create_db.sh

```diff
@@ -15,18 +15,20 @@ docker volume rm wordle_dbdata
 echo "Now running the postgres container"
 container_id=$(docker run -it -d  -e POSTGRES_USER=${pg_user} -e POSTGRES_DB=${pg_db} -e POSTGRES_PASSWORD=${pg_pass} -v $PWD:/opt/sql -v wordle_dbdata:/var/lib/postgresql/data postgres:13) 
 
-
+while ! docker exec ${container_id} pg_isready; do
```

Comment:
I recommend writing tests always with the full syntax `[[ test ]]`. Bash has loads of corner cases and it's better to be very strict about such things. Have you checked that the exit status of `docker exec` is the same of `pg_ready`? The rest of the loop is correct.